batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,56 @@
+package com.lftechnology.batch7crud.controller;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by pratishshr on 1/19/16.
+ */
+public class CustomHttpServlet extends HttpServlet {
+
+    private static final String ERROR_PAGE = "/WEB-INF/views/error.jsp";
+    private static final String MESSAGE = "message";
+
+    public void show404(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: javax.servlet.ServletException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)